common/src/test/java/com/sequenceiq/cloudbreak/logger/format/CustomJsonLayoutTest.java

```diff
@@ -0,0 +1,33 @@
+package com.sequenceiq.cloudbreak.logger.format;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import ch.qos.logback.classic.spi.LoggingEvent;
+
+public class CustomJsonLayoutTest {
+
+    private CustomJsonLayout underTest;
+
+    @Before
+    public void setUp() {
```

Comment:
Would you mind to create tests based on the size message bigger than the 12KB limit or not?